uint8 take_photo=1
uint8 start_video=2
uint8 stop_video=3

int16 yaw      #deg * 0.1
int16 pitch    #deg * 0.1
int16 roll       #deg * 0.1

uint8 action
